config_ext.h

```diff
@@ -78,12 +74,26 @@
 #undef SQLITE_OMIT_VACUUM
 #undef SQLITE_OMIT_VIEW
 #undef SQLITE_OMIT_VIRTUALTABLE
-#define SQLITE_OMIT_WAL                      1
 #undef SQLITE_OMIT_WSD
 #define SQLITE_OMIT_XFER_OPT                 1
-#define SQLITE_PERFORMANCE_TRACE             1
 /* #define SQLITE_OMIT_COMPLETE              1 */
 /* #define SQLITE_OMIT_SUBQUERY              1 */
 /* #define SQLITE_OMIT_DATETIME_FUNCS        1 */
 /* #define SQLITE_OMIT_FLOATING_POINT        1 */
-#define SQLITE_COUNTOFVIEW_OPTIMIZATION      0
+
+#define SQLITE_OMIT_JSON                     1
+#define SQLITE_OMIT_INTROSPECTION_PRAGMAS    1
+#define SQLITE_OMIT_COMPLETE                 1
+#define SQLITE_OMIT_DESERIALIZE              1
+#define SQLITE_DISABLE_PAGECACHE_OVERFLOW_STATS 1
+#define SQLITE_ENABLE_NULL_TRIM              1
+#define SQLITE_ENABLE_PREUPDATE_HOOK         1 // This may be interesting to implement the update logic
```

Comment:
I was reading about this too but I'm afraid not, as the hook cannot do anything on the database (nor change what has been asked to do). Here is an example/question which may be similar to your idea that should not be feasible with the hook indeed: https://stackoverflow.com/questions/37545766/sqlite-pre-update-hook-re-entrant

Reply:
what a pity!

Removed